src/views/mixed-content/index.js

```diff
@@ -10,10 +18,17 @@ import React from 'react';
  * @return {ReactNode} Mixed Content View
  */
 const MixedContent = () => {
-  // TO-DO(ernestognw): To fill with mixed-content data graphs
   return (
     <div data-testid="mixed-content">
-      <p>Mixed Content</p>
+      <h1>Mixed Content</h1>
+      <h2>Government websites</h2>
+      <p>Websites were considered from a government if they have
```

Comment:
I think this string is weirdly placed because its relevant only to the first visualization, maybe we could move it at the bottom of the page as a note or include it in the TopGovernmentWebsites card?

Reply:
@SofiaVega, I thin this can be fixed on #9 

I'm just linking the comment to take into consideration